Sources/Settings/Settings.swift

```diff
@@ -3,6 +3,14 @@ The namespace for this package.
 */
 public enum Settings {}
 
+/// A typealias for this package's namespace to solve the conflict with [SwiftUI.Settings](https://developer.apple.com/documentation/swiftui/settings)
```

Comment:
This is currently the right behavior to reference Apple's framework symbol in swift-docc.

It will show as a link and jump to the right page both on Web and Xcode's documentation Window.